front/src/components/header/Header.tsx

```diff
@@ -82,8 +81,8 @@ const Header = ({ isLoggedIn, setIsLoggedOut }: Props) => {
   }) => (
     <Link onClick={closeHeader} to={linkTo}>
       <Text
-        borderBottom={isActiveRoute ? '3px solid teal' : ''}
-        color={isActiveRoute ? 'teal.600' : ''}
+        borderBottom={isActiveRoute ? '1px solid #0099DB' : ''}
+        color={isActiveRoute ? '#034A85' : '#0099DB'}
```

Comment:
Let me know if you think all these colors should be stored elsewhere, they're just the few shades of blue from the logo, brought over.

Reply:
There is a `theme.js` file but I haven't messed with it much. I might try adding the colors there later but not necessary for now 👍